susemanager-sync-data/additional_products.json

```diff
@@ -1,4 +1,91 @@
 [
+        {
```

Comment:
This file should come from sumatoolbox. I think it would be better to copy it to an issue in gitlab and we work on it there.
For now it is better to remove it from this PR

Reply:
Why do we have a copy in the Uyuni repository?

Reply:
We have this package from the old times where we shipped a channels.xml file.
So this package was used for shipping data about products and use the tooling we have for packages comming from this repo.
Mid term goal is, that we do not need to ship these files anymore and get all the data from SCC.